src/sider/manglendemedvirkning/forhandsvarsel/ForhandsvarselSendt.tsx

```diff
@@ -0,0 +1,21 @@
+import { VurderingResponseDTO } from "@/data/manglendemedvirkning/manglendeMedvirkningTypes";
+import dayjs from "dayjs";
+import React from "react";
+import ForhandsvarselBeforeDeadline from "@/sider/manglendemedvirkning/forhandsvarsel/ForhandsvarselBeforeDeadline";
+import { ForhandsvarselAfterDeadline } from "@/sider/manglendemedvirkning/forhandsvarsel/ForhandsvarselAfterDeadline";
+
+interface Props {
+  sisteVurdering: VurderingResponseDTO;
+}
+
+export default function ForhandsvarselSendt({ sisteVurdering }: Props) {
+  const isForhandsvarselExpired = dayjs(
+    sisteVurdering?.varsel?.svarfrist
+  ).isBefore(dayjs(new Date()), "date");
```

Comment:
Hvis du lager noe funksjon på dette for aktivitetskrav og arbeidsuførhet, så kan jeg bruke det også, @andersrognstad 👍🏼